### PR TITLE
Update ug setcrs

### PR DESCRIPTION
**Pull request by saiutkarsh33**
Solves #139 and  #167 and #211

### CODE REVIEW

**Comment by tituschewxj** [comment]
@saiutkarsh33 Could you rebase your PR, as there are merge conflicts?

File: docs/UserGuide.md [hunk 1]
```diff
@@ -180,14 +180,21 @@ The quick reference is meant for **fast and reliable** lookup of commands and th
...
+Course is shown at the top of the window.
+
+Format: `setcrs COURSE_code`
+
+<img src="images/ug/setcrs_2101.png" alt="Setcrs 2101" class="rounded-image" width="800px"/>
+
+<img src="images/ug/setcrs_successful.png" alt="Setcrs 2101" class="rounded-image" width="800px"/>
 
-Duplicate course are not allowed.
 Courses are case-insensitive.
```
Comment (tituschewxj) [inline docs/UserGuide.md:193]: We can consider using boxes to add whitespace, which improves readability.
```suggestion
<box type="info" light>

Course codes are case-insensitive.
</box>

```

File: docs/UserGuide.md [hunk 2]
```diff
@@ -180,14 +180,21 @@ The quick reference is meant for **fast and reliable** lookup of commands and th
...
 Course code should follow the format "XX1234Y", Y is optional.
 
+<img src="images/ug/setcrs_error.png" alt="Setcrs Error" class="rounded-image" width="800px"/>
```
Comment (tituschewxj) [inline docs/UserGuide.md:196]: We can put this into a separate box, and add a line to describe the image.
```suggestion
<box type="warning" light>

Course code should follow the format "XX1234Y", Y is optional.

When an invalid course code is entered, the follow error message appears:

<img src="images/ug/setcrs_error.png" alt="Setcrs Error" class="rounded-image" width="800px"/>
</box>
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -200,6 +207,7 @@ Format: `addstu n/NAME nn/NUSNET [p/PHONE] [e/EMAIL] [m/MAJOR] [t/TAG]…​`
 * Add a student with the given details.
 * The name and nusnet id must be provided. And nusnet id must be unique.
 * All the remaining fields are optional. If not provided, a placeholder value will be used.
+* Name cannot be empty or spaces only, contain only alphabets and cannot have double spaces
```
Comment (tituschewxj) [inline docs/UserGuide.md:210]: We can put this in a box to improve readability:
```suggestion

<box type="warning" light>

Name cannot be empty or spaces only, contain only alphabets and cannot have double spaces 
</box>
```

**Review by tituschewxj — CHANGES_REQUESTED** [review]
I have a few nits. 

The changes you made in your previous PRs are duplicated here. Hence we should only focus on the issues in this PR. There will likely be a need to rebase.

>[!IMPORTANT]
>This PR can only be merged after #258 is merged!

**Review by belligerentbeagle — CHANGES_REQUESTED** [review]
Overall LGTM, just one typo! Can also consider Titus' above recommendation on the light (i think now seamless) box

**Review by tituschewxj — APPROVED** [review]
LGTM! Except that this doesn't really resolve: https://github.com/AY2324S2-CS2103T-F13-1/tp/issues/167, to resolve this we should also have a successful example where the course name is set. We can do this in another PR.

**Comment by belligerentbeagle** [comment]
Hi @saiutkarsh33 , do also consider adding a note that this app would not support courses with more than 2 prefixes since it is meant for CS module TAs. 
See
"Make it clear that the courses being taught are CS courses. See [PE-D][Tester B] Unable to set course code for certain formats of course codes.  #164" in #252

**Comment by Anant1902** [comment]
> Hi @saiutkarsh33 , do also consider adding a note that this app would not support courses with more than 2 prefixes since it is meant for CS module TAs. This would help you defend against an issue like #164 See "Make it clear that the courses being taught are CS courses. See [PE-D][Tester B] Unable to set course code for certain formats of course codes. #164" in #252

@saiutkarsh33 Bumping this hahahah

**Comment by tituschewxj** [comment]
I'll force merge this and fix the issues of duplicated `Course is shown...`, as I don't want this to have more merge conflicts.

**Pull request merged**